jenkins/Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+pipeline {
+    agent {
+        label 'tsa'
+    }
+    post {
+        always{
+            echo 'Cleaning up workspace'
+            deleteDir()
+        }
+    }
+    stages {
+        stage('Build') { 
+            steps {
+	    	sh '''#!/bin/bash
+     	    	cd src
+	    	make
+                ''' 
+            }
+        }
+        stage('Test') { 
+            steps {
+                sh '''
+		./src/ffixed2free
```

Comment:
I think the tests should do test the output, not only run it. 
you could have a collection of small examples to test, together with a reference, which is the expected output. Then you could have a test.py that would basically run all the examples and compare the output produce against the reference (with a simple diff)

Reply:
Added FORTRAN 77 example file for testing, wich is now use in the 'Test' stage of Jenkins.